package.json

```diff
@@ -101,7 +101,7 @@
     "libphonenumber-js": "1.9.34",
     "lint-staged": "10.5.4",
     "lodash": "^4.17.15",
-    "lodestar-app-element": "urfit-tech/lodestar-app-element#master",
+    "lodestar-app-element": "urfit-tech/lodestar-app-element#49c2409933749884eed3615de0aae493f305f8d8",
```

Comment:
#master

Reply:
The hash is pushed because the element is be developed. After discussion, just note it in the description.